.github/workflows/main.yml

```diff
@@ -24,6 +24,7 @@ jobs:
           cd linux
           ./autogenerate.sh
       - name: Build docker image
+        if: contains(matrix.pg, 'pg12') || (!contains(matrix.os, 'ubuntu2004') && contains(matrix.pg, 'pg11'))
```

Comment:
It took me a while to figure out the check here. Do I read that correctly as PSQL 12 for any platform + PSQL 11 for any platform except Ubuntu 20.04?

Possibly this logic could be moved alongside the matrix metadata above using the [exclude](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-configurations-from-a-matrix) keyword to ease maintenance?

Reply:
You read it correctly. Exclude will make things cleaner.